site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -223,7 +229,18 @@ The number of rules available after that depends on how many rules are enabled b
 
 #### Dynamic and session rules {: #dynamic-session-rules }
 
-The limits applied to dynamic and session rules are simpler than static rules. The total number of both cannot exceed 5000. This is called the [`MAX_NUMBER_OF_DYNAMIC_AND_SESSION_RULES`](#property-MAX_NUMBER_OF_DYNAMIC_AND_SESSION_RULES).
+An extension can have up to 5000 session rules. This is exposed as the
+[`MAX_NUMBER_OF_SESSION_RULES`](#property-MAX_NUMBER_OF_SESSION_RULES).
+
+For dynamic rules, up to 30,000 rules are permitted, exposed as the
+[`MAX_NUMBER_OF_DYNAMIC_RULES`](#property-MAX_NUMBER_OF_DYNAMIC_RULES). Included in this set of
```

Comment:
Do I understand this right, that the 5000 count towards the 30000 limit?

Reply:
You do :)